frontend/src/app/modules/availability/availability-page/availability-page.component.html

```diff
@@ -5,7 +5,7 @@
             <button mat-flat-button class="new-meeting-button" routerLink="/new-meeting">+ New Meeting</button>
             <button mat-flat-button color="primary" routerLink="/availability/new" class="new-availability-button">+ New Availability Slots</button>
         </div>
-        <div *ngIf="userPersonalAndTeamSlots">
+        <div>
```

Comment:
Don't we need to check for null?
I believe we will have console error without check for null

Reply:
Fixed